netlify.toml

```diff
@@ -1,6 +1,5 @@
 [Settings]
 ID = "d2668848-c2a6-411f-b884-004faee1942f"
 [build]
-functions = "./.netlify/functions"
-# Specify is-core-module/package.json as an external dependency
-ignore = "is-core-module/package.json"
\ No newline at end of file
+  command = "echo \"export const COMMIT_HASH = '$COMMIT_REF';\" > src/commit-hash.ts"
```

Comment:
This is a hack to access the current commit hash only during Netlify build time, and saving it to be loaded in the runtime later.